docs/BUILD-portable.md

```diff
@@ -0,0 +1,37 @@
+# Build Guide (WIP)
+
+The information currently listed here should be enough to be able to build the picoTracker "portable" if you've done projects like these before. 
+
+## Components
+
+Here are linked examples of sources for the components, but they can be purchased in many different places.
+
+If you go with your own case some of these things can be relaxed. i.e: can use any ili9341 display as long as it's 320x240, can use any MIDI ports, could use the SD card slot that comes with the display (thou I may decide in the future to use SDIO exclusively and that would break your build), etc.
+
+* [PCBs and PCB SMT assembly](https://github.com/democloid/picoTrackerPortable-pcb) can be done at JLCPCB
+* [ILI9341 2.8" NO TOUCH NO PCB display](https://www.aliexpress.com/item/1005003732740869.html)
+* [3.7V 700mAh 802535 Battery](https://www.aliexpress.com/item/1005005279230561.html)
+* [9 x Cherry style switches](https://www.aliexpress.com/item/1005006005139724.html)
+* [M3 & M2 screws](https://www.aliexpress.com/item/32810852732.html)
+* [*Optional* Keycaps](https://www.aliexpress.com/item/1005003849297167.html)
+* [3D printed case](https://github.com/democloid/picoTrackerPortable-case)
+* [*Optional* 3D printed keycaps](https://www.thingiverse.com/thing:5180641)
+
+![Components](img/components-portable.jpg)
+
+For those that prefer getting a prebuilt version of the kit there is [one currently available](https://www.tindie.com/products/maksmakes/picotracker-kit/).
```

Comment:
Do we want to add this here is we are not committing to have them continuously?

Reply:
yes good point. I'll take it out, as Im really not sure about continuing to make the kits, can always add it in if someone else decided to commit to making them available long term